docker/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Copyright 2022, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+case "$(uname -m)" in
+    arm64)
```

Comment:
This does not really detect Apple M1 or its distribution, right? So e,g, on a RasPI LInux aarch64 this may fail then.  Maybe add a note here in the source at least, that this is expected and could get more improvement when somebody runs into this.

Reply:
Suggest adding   test for `"$(uname -s)" = "Darwin"`
Maybe:
```
case "$(uname-s)-$(uname-n)"  in
 Darwin-arm64)
   ...
 ```

Reply:
It should actually work on any Arm docker that is new enough to have `buildx`. The problems are not specific to the M1, it's just that docker wants to build on the native architecture by default and M1 happens to be popular. So if you have RasPI Linux aarch64 with docker buildx installed, I'd expect this to work.